spec/factories/event.rb

```diff
@@ -0,0 +1,12 @@
+FactoryBot.define do
+  factory :event do
+    name        'GURU-SP'
+    description '50th meeting'
+    association :user
+    occurs_at   { 1.month.from_now }
+
+    trait :ruby do
+      after(:create) { |event| event.tag_list.add("ruby"); event.save }
+    end
+  end
+end
```

Comment:
@frankyston minor change here, please: https://stackoverflow.com/a/5813359

Reply:
Done. 2660fd95c3df39691d91e2cfdffe858821ef993c

Reply:
I still the "no newline at end of file" issue